mlrun/runtimes/nuclio/api_gateway.py

```diff
@@ -27,9 +27,10 @@
 from mlrun.platforms.iguazio import min_iguazio_versions
 from mlrun.utils import logger
 
+from ...common.constants import MlrunInternalLabels
 from .function import get_fullname, min_nuclio_versions
 
-PROJECT_NAME_LABEL = "nuclio.io/project-name"
+PROJECT_NAME_LABEL = MlrunInternalLabels.nuclio_project_name
```

Comment:
Don't do the change here, have everywhere that used this `PROJECT_NAME_LABEL` and replace with `MlrunInternalLabels.nuclio_project_name`, then remove this.